docker-compose.dist.yml

```diff
@@ -13,37 +13,36 @@ services:
         environment:
             - APP_ENV=dev
             - APP_DEBUG=1
-            - DB_DRIVER=pdo_pgsql
```

Comment:
Please remove first commit as it shouldn't be part of this PR.
The rest LGTM.

Reply:
Done